packages/replicache/src/persist/collect-idb-databases.ts

```diff
@@ -286,14 +299,21 @@ export function deleteAllReplicacheData(
  * `[false]`. Otherwise we return `true` and an array of the clientIDs to
  * remove.
  */
-function gatherPendingMutationsInClientGroups(
+function canDatabaseBeCollectedAndGetDeletedClientIDs(
```

Comment:
there is similar logic in client-group-gc that we should update as well.  https://github.com/rocicorp/mono/blob/3399ecc55c86dfbd8a63e247b20c2da608f1c221/packages/replicache/src/persist/client-group-gc.ts#L52

Reply:
#3719